docs/content/projects/BuildTrace.md

```diff
@@ -9,25 +9,83 @@ This plugin collects tasks execution time in a trace event format.
 
 ## Getting started
 
-Apply the plugin in the root `build.gradle` file:
+Apply the plugin in the root buildscript:
 
-```groovy
+```kotlin
 plugins {
     id("com.avito.android.build-trace")
 }
+
+buildTrace {
+    enabled.set(true)
+}
 ```
 
 --8<--
 plugins-setup.md
 --8<--
 
-Run a build with `--profile` argument. You will get a message in a log:
+Run a build. You will get a message in a log:
 
 ```log
-Build trace: <path to the project>/outputs/trace/build.trace
+Build trace: <path to the project>/outputs/avito/build-trace/build.trace
```

Comment:
```suggestion
Build trace: <path to the project>/outputs/<project name>/build-trace/build.trace
```

Reply:
The plugin works in a root project. It's not a project name but a company name.
This is to mitigate probable clashes with other "outputs".

Reply:
Removed "avito" prefix